.taskmaster/error-handler.py

```
@@ -0,0 +1,350 @@
+            
+            if os.path.exists(pid_file):
+                try:
+                    with open(pid_file, 'r') as f:
+                        pid = int(f.read().strip())
+                    
+                    process = psutil.Process(pid)
+                    running = process.is_running()
+                    if running:
+                        cpu_usage = process.cpu_percent()
+                        memory_usage = process.memory_percent()
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)